src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+
+import DotSlider from "./DotsSlider/index";
+
+import { StyledButton } from "./styles";
+
+import { carouselDataMapped } from "./utils";
+
+import { ICarousel } from "./types";
+
+const Carousel: React.FC<ICarousel> = ({ click, carouselData }: ICarousel) => {
+    const [activeTabIndex, setActiveTabIndex] = useState(0);
+
+    const changeActiveTabHandler = (index: number) => {
+        return setActiveTabIndex(index);
+    };
+
+    return (
+        <main id="carousel">
+            {carouselDataMapped(carouselData, activeTabIndex)}
+            <DotSlider
+                activeTabIndex={activeTabIndex}
+                click={changeActiveTabHandler}
+                carouselData={carouselData}
+            />
+            {activeTabIndex === carouselData.length - 1 ? (
+                <StyledButton onClick={() => click()}>Continuar</StyledButton>
```

Comment:
Habría que bajar el cierre `</StyledButton>` a la linea de abajo

Reply:
Creo que esto lo determina el prettier, tipo, correr el prettier y lo va a ordenar bien bonito

Reply:
![Done](https://user-images.githubusercontent.com/64698390/104970685-1f1a9100-59cb-11eb-8c5c-7e0ecce381e6.gif)

... Yo ser indio !! prettier no hacer trabajo... prettier wanted it to look like this Btw